scripts/qa_crowbarsetup.sh

```diff
@@ -2632,6 +2632,8 @@ function oncontroller_testsetup()
             SLE11SP3-x86_64-cfntools | tee glance.out
         imageid=`perl -ne "m/ id [ |]*([0-9a-f-]+)/ && print \\$1" glance.out`
         crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
+        # test if is cnftools image prepared for tempest
+        wait_for 300 5 '[ "active" == $( openstack image show -f value -c status $imageid) ]' "prepare cnftools image"
```

Comment:
also I think, we have similar code for docker and the normal SLE image - maybe time for some refactoring to do the test-if-there ; upload or show > glance.out ; wait_for active